OtripleS.Web.Api/Services/Classrooms/IClassroomService.cs

```diff
@@ -13,6 +13,7 @@ namespace OtripleS.Web.Api.Services.Classrooms
 	public interface IClassroomService
 	{
 		ValueTask<Classroom> CreateClassroomAsync(Classroom classroom);
+        ValueTask<Classroom> GetClassroomById(Guid classroomId);
```

Comment:
fix your indentations

Reply:
I ran code cleanup to fix these just now, there were several other files affected by the cleanup (ordering of imports in most of them).